store/src/java/com/zimbra/cs/mailbox/MailItem.java

```diff
@@ -986,6 +986,10 @@ public long getDate() {
      *  content was modified.  For immutable objects (e.g. received messages),
      *  this will be the same change ID as when the item was created. */
     public int getSavedSequence() {
+        return (int) mData.modContent;
```

Comment:
Same question here, what's the expected behavior on loss if modContent is bigger than int max? Any reason we're not changing the method signatures all the way through?

Reply:
Its being used by NG module. If we change the return type from int to long, their code breaks since long can not fit into int. Eventually, they should use the new getSavedSequenceLong() in their code. But, this is to avoid any immediate functionality errors.